packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -1101,15 +1101,16 @@ describe('network stubbing', { retries: 2 }, function () {
         }).visit('/foo')
       })
 
-      it('can timeout in request handler', {
-        defaultCommandTimeout: 50,
-      }, function (done) {
+      it('can timeout in request handler', function (done) {
         cy.on('fail', (err) => {
+          Cypress.config('defaultCommandTimeout', 5000)
           expect(err.message).to.match(/^A request callback passed to `cy.route2\(\)` timed out after returning a Promise that took more than the `defaultCommandTimeout` of `50ms` to resolve\./)
```

Comment:
sometimes this can take more than 50ms due to command log lag :upside_down_face: